README.md

```diff
@@ -35,6 +35,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 
 ## A - C
 
+- [Aayulogic Pvt. Ltd.](https://aayulogic.com/) | 4th Floor, Ganapati Bhawan, Min Bhawan Main Road | Software Solutions including, but not limited to, web development, web desigining, web/domain hosting, cms etc
```

Comment:
Hello @annshress, thank you for the pull request. 

Can you shorten down the location to `Location, City`.